.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -0,0 +1,149 @@
+name: Bug report
+description: Report a bug with the WordPress block editor or Gutenberg plugin
+title: "<title>"
+body:
+
+- type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report! If this is a security issue, please report it in HackerOne instead: https://hackerone.com/wordpress
+
+- type: checkboxes
+  attributes:
+    label: Is there an existing issue for this?
+    description: Please heck if the bug has already been reported by searching https://github.com/WordPress/gutenberg/issues.
+    options:
+    - label: I have searched the existing issues
+      required: true
+      
+- type: checkboxes
+  attributes:
+    label: Have you tested with the latest version of WordPress and/or Gutenberg?
+    description: Please make sure you are using the latest versions before reporting.
+    options:
+    - label: I have tested against the latest versions.
+      required: true
+
+- type: checkboxes
+  attributes:
+    label: Have you tested with the latest version of WordPress and/or Gutenberg?
+    description: Please make sure you are using the latest versions before reporting.
+    options:
+    - label: I have tested against the latest versions.
+      required: true
+      
+- type: checkboxes
+  attributes:
+    label: Have you tried deactivating all plugins except Gutenberg?
+    description: Please make sure you have ruled out plugin conflicts before reporting.
+    options:
+    - label: I have tested with all plugins deactivated. 
+      required: true
+      
+- type: checkboxes
+  attributes:
+    label: Have you tried replicating the bug using a default theme e.g. Twenty Twenty?
+    description: Please make sure you have confirmed it's not a theme specific problem.
+    options:
+    - label: I have tested with a default theme.
+      required: true
+
+- type: checkboxes
+  attributes:
+    label: Have you tried replicating the bug using a default theme e.g. Twenty Twenty?
+    description: Please make sure you have confirmed it's not a theme specific problem.
+    options:
+    - label: I have tested with a default theme.
+      required: true
```

Comment:
Makes me think there's too much checkboxes to create a bug report. Just want to make sure we want that or if we can leave the most important one?

For instance, there's a check about testing with the latest Gutenberg version and there's a Gutenberg version selector in the end. It's kind of contradictory.

Reply:
Agreed - I was trying to match as closely as possible what was in the prior form to start but let's simply this :) I'll update and remove the latter pieces.

Reply:
Thinking about this more, decided to keep the fields where people can enter versions as it's more important that we know what versions someone is using rather than relying on the checkbox since we might not get to a report for some time.